.copier-answers.yml

```diff
@@ -19,6 +20,6 @@ repo_description: 'TODO: add repo description.'
 repo_name: knowledge
 repo_slug: knowledge
 repo_website: https://github.com/OCA/knowledge
-travis_apt_packages: []
-travis_apt_sources: []
+use_pyproject_toml: true
```

Comment:
Use false here in order to remove that files

Reply:
> Use false here in order to remove that files

Done. Thank you for the hint!

Reply:
The files are still there. They are a leftover.